src/addons/analog.cpp

```diff
@@ -13,6 +13,11 @@
 #define ANALOG_CENTER 0.5f // 0.5f is center
 #define ANALOG_MAX 1.0f    // 1.0f is max
 
+uint16_t adc_1_x_center = 0;
```

Comment:
Is there a particular reason why these variables are placed in the global scope? If at all possible these should be member variables of `AnalogInput`.